packages/components-sketch/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
do we want this file checked in to git?

Reply:
could be
depends if we want to stay IDE agnostic^^

Reply:
..and again, since handlebar templates will hopefully replaced, this can go